src/showcase/index.md

```diff
@@ -42,29 +46,51 @@ toc: false
         {% for case in site.data.showcases %}
             {% capture modulo3 %}{{ forloop.index0 | modulo:3 }}{% endcapture %}
             {% if modulo3 == '0' %}
-                <div class="showcase__apps__cards text-left card-deck">
+                <div class="showcase__apps__cards text-left card-deck flex-column flex-lg-row">
             {% endif %}
                     <div class="card">
+                        <div class="card-header">
+                            {% if case.logo_src %}
+                                {% asset '{{ case.logo_src }}' class='showcase__apps__logo' %}
+                            {% endif %}
+                        </div>
                         <div class="card-body">
                             <h3>{{ case.name }}</h3>
                             <p>{{ case.description }}</p>
                         </div>
                         <div class="card-footer">
-                            <a class="btn btn-link" href="{{ case.learn_more_link }}">Learn more</a>
+                            {% if case.learn_more_link %}<a href="{{ case.learn_more_link }}">Learn more</a>{% endif %}
+                            {% if case.play_store_link or case.app_store_link %}
```

Comment:
nit: Btw, _generally_, you should use `-%}` to close a Liquid tag. The hyphen instructs Liquid to eat up (in this case trailing) whitespace.